lib/raygun/client.rb

```diff
@@ -144,7 +152,12 @@ def filter_custom_data(env)
       def build_payload_hash(exception_instance, env = {})
         custom_data = filter_custom_data(env) || {}
         tags = env.delete(:tags) || []
-        tags << rack_env if rack_env_present?
```

Comment:
Suggest that this would work as `rails_env || rack_env` without the need for the `_present?` method.